src/components/dialogs/notificationDialogUtils/NotificationCollaborationManagerRevokeDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { get } from 'lodash-es';
+import { notificationSchema } from '../../../constants/notificationSchema';
+import NotificationDetailsDialog from '../NotificationDetailsDialog';
+
+export default function NotificationCollaborationMangerRevokeDialog({
+  notification,
+  onClose,
+  open,
+}) {
+  const notificationType = notification?.message_type;
+  const currentNotificationSchema = get(
+    notificationSchema,
+    notificationType,
+  );
+  const avaialableButtons = [];
```

Comment:
typo

Reply:
actually i'm noticing there are several of these, and also several `error=""` below. just do a scan for them please and i'll stop sending you emails for each of them :)

Reply:
See above.

Reply:
Done